src/client/components/menu/Menu.component.js

```diff
@@ -0,0 +1,147 @@
+import React from 'react';
+import { BrowserRouter, Link } from 'react-router-dom';
+import './Menu.styles.css';
+import faUser from '../../assets/images/user-login.png';
+import faHeart from '../../assets/images/favorite-icon.png';
+import faShoppingCart from '../../assets/images/shopping-cart.png';
+
+export const Menu = () => {
+  return (
+    <menu>
+      <header>
+        <BrowserRouter>
```

Comment:
BrowserRouter should be only used once at the top level component of the application which app.js (check line 22)